app/src/main/java/com/sun_asterisk/comics_01/data/source/remote/fetchjson/ParseDataWithJson.java

```diff
@@ -0,0 +1,90 @@
+package com.sun_asterisk.comics_01.data.source.remote.fetchjson;
+
+import android.util.Log;
+import com.sun_asterisk.comics_01.data.model.Comic;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class ParseDataWithJson {
+    private static final int TIME_OUT = 15000;
+    private static final String METHOD_GET = "GET";
+
+    public String getJsonFromUrl(String urlString) throws Exception {
```

Comment:
sử dụng static để tránh việc tạo đối tượng mỗi lần muốn gọi.
```suggestion
    public static String getJsonFromUrl(String urlString) throws Exception {
```